package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bandwidth/mfa",
-  "version": "1.1.1",
+  "version": "1.2.0",
```

Comment:
Can we change this to `2.0.0` since the SDK is having a breaking change with the removal of the required fields?

Reply:
updated